docs/general/administration/hardware-selection.md

```diff
@@ -0,0 +1,125 @@
+---
+uid: admin-hardware-selection
+title: Hardware Selection
+---
+
+# Selecting Appropriate Hardware
+
+This page covers what you need to know in order to select appropriate hardware for a Jellyfin server and take full advantage of its features (ex. Hardware Acceleration).
+
+## TL;DR
+
+For a Jellyfin server, we recommend the following:
+
+### Normal Server From Scratch
+
+- CPU: 4 Threads
+  - Please Avoid 11th Gen or older Intel Atom and Celeron / Pentium N / J series CPUs.
```

Comment:
Where is this coming from?

Reply:
4 threads is what I think should be able to comfortably handle most software decoding tasks in legacy formats.
Atom, Celeron/Pentium N/J series was excluded because they are low powered parts that may struggle in some tasks but correct me if I'm wrong here.

Reply:
This line no longer exists in PR and I will close this